src/status_im/transport/shh.cljs

```diff
@@ -133,14 +133,22 @@
                                  (on-success chat-id sym-key sym-key-id))
                    :on-error   log-error}))))
 
+(defn add-new-sym-key [{:keys [web3 sym-key on-success]}]
+  (add-sym-key {:web3       web3
+                :sym-key    sym-key
+                :on-success (fn [sym-key-id]
+                              (on-success sym-key sym-key-id))
+                :on-error   log-error}))
+
+(re-frame/reg-fx
+ :shh/add-new-sym-keys
+ (fn [keys]
```

Comment:
Is this working ? Isn't it suppose to receive a map as parameter ? and add-new-sym-key as well ?

Reply:
Here in PR it receives vector always. `add-new-sym-key ` receives map. Basically messaging is working in this PR, so i believe it's not broken.

Reply:
it is broken but we use public key for messages. try updating a contact

Reply:
maybe it is not broken but it must be renamed args because keys is just plain wrong this is not keys